README.md

```diff
@@ -25,7 +25,9 @@ Note this package is in preview and it will not work with your Tecton installati
 
 ## Changelog
 
-- 0.0.3 Support `allow_overwrite` setting in the operators
+- 0.1.0 Added 2 new operators to support triggering Feature Table ingestion jobs 
+
+- 0.0.3 Supported `allow_overwrite` setting in the operators
```

Comment:
thanks for fixing this. I think the proper phrasing is probably `Added support for`